packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx

```diff
@@ -89,11 +90,11 @@ export const SPAPropertyDetailPage = (): JSX.Element => {
                 </TabTitleIcon>
                 <TabTitleText>Settings</TabTitleText>
               </>
-            }
+         
             aria-label="SPA-detailed-Settings"
           >
             {openTab === 3 && <Settings />}
-          </Tab>
+          </Tab> */}
```

Comment:
Please remove these codes

Reply:
This is a temporary fix, hence had added a NOTE and commented it